### PR TITLE
Add sudo-edit package to recipes.

### PR DESCRIPTION
**Pull request by nflath**

### CODE REVIEW

File: recipes/sudo-edit [hunk 1]
```diff
@@ -0,0 +1,3 @@
+(mv-shell
```
Comment (syohex) [inline recipes/sudo-edit]: Wrong package name. `mv-shell` -> `sudo-edit`

File: recipes/sudo-edit [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+ :url "https://github.com/nflath/sudo-edit.git"
```
Comment (syohex) [inline recipes/sudo-edit]: To use `:fetcher github` and `:repo` attribute is better than `:fetcher git` + `:url` attribute.

**Comment by purcell** [comment]
Thanks. Looks good, but please don't do this:

```el
(global-set-key (kbd "C-c C-r") 'sudo-edit-current-file)
```

The user should choose any global keybindings. Maybe suggest the above in the `Commentary` section of the file.

If you want to make the code more robust, you might consider handling the case where `buffer-file-name` is nil. And maybe `(concat "/sudo:root@localhost:" x)` should be extracted as a little helper function to avoid repeating the magic string.

When I commit this I'd squash the two commits to keep our history clean. If you have a moment to that, it would make my life easier.

**Comment by nflath** [comment]
I've fixed the keybinding and squashed the commit.

**Comment by purcell** [comment]
I don't see any change re. the keybinding. Did you maybe forget to push changes?

**Comment by nflath** [comment]
Yep.  Pushed now.

**Comment by purcell** [comment]
Yay, thanks.

**Pull request merged**